bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/common/registry/AbstractRegistry.java

```diff
@@ -118,6 +118,13 @@ public void added(Provider<E> provider, E element) {
         Collection<E> elements = elementMap.get(provider);
         if (elements != null) {
             try {
+                K uid = element.getUID();
+                if (uid != null && get(uid) != null) {
+                    logger.warn(String.format(
```

Comment:
Is there a special reason why you use `String.format` and not simply use parameterized logging?